src/components/Callout/Callout.md

```diff
@@ -0,0 +1,21 @@
+### Notes for use
+
+Callout is simply exported from Fabric, with some minor YamUI styling. It is meant to be used as a utility component by other YamUI components like Hovercard and Tooltip, you probably don't need to use it directly in your application UIs.
+https://developer.microsoft.com/en-us/fabric#/components/callout
+
+### Examples
+
+Basic usage:
+
+```js
+const visible = true;
+
+<div>
+  <span className="myTarget">Target</span>
+  {visible && (
+    <Callout target=".myTarget">
```

Comment:
String refs are getting removed soonish: https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs

I wouldn't use them in an example, cause people will copy it.

Reply:
done